src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -196,15 +195,15 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
   const permissionConsentTypeLabel = (permissionInAllPrincipal : boolean) : JSX.Element => {
     if(permissionInAllPrincipal){
       return (
-        <div style={{textAlign: 'left', paddingLeft: '10px'}}>
-          <Label>{translateMessage('AllPrincipal')}</Label>
+        <div style={{textAlign: 'center', paddingLeft: '10px'}} aria-label='AllPrincipal'>
```

Comment:
The aria-label could also benefit from translation :)

And the styling is repeated.